src/diffusers/models/unets/unet_2d_condition.py

```diff
@@ -1185,7 +1175,13 @@ def forward(
             cross_attention_kwargs["gligen"] = {"objs": self.position_net(**gligen_args)}
 
         # 3. down
-        lora_scale = cross_attention_kwargs.get("scale", 1.0) if cross_attention_kwargs is not None else 1.0
+        # we're popping the `scale` instead of getting it because otherwise `scale` will be propagated
+        # to the internal blocks and will raise deprecation warnings. this will be confusing for our users.
+        if cross_attention_kwargs is not None and "scale" in cross_attention_kwargs:
+            lora_scale = cross_attention_kwargs.pop("scale", 1.0)
```

Comment:
made change based on @BenjaminBossan 's suggestion https://github.com/huggingface/diffusers/pull/7338/files#r1528407445
```suggestion
        lora_scale = cross_attention_kwargs.pop("scale", 1.0)
```